src/common/wal/src/config/kafka/datanode.rs

```diff
@@ -40,6 +40,11 @@ pub struct DatanodeKafkaConfig {
     /// The kafka topic config.
     #[serde(flatten)]
     pub kafka_topic: KafkaTopicConfig,
+    pub create_index: bool,
+    #[serde(with = "humantime_serde")]
+    pub dump_index_interval: Duration,
+    #[serde(with = "humantime_serde")]
+    pub index_checkpoint_interval: Duration,
```

Comment:
Maybe we can merge dump_index and index_checkpoint?

Reply:
Resolved in [73ac3b1](https://github.com/GreptimeTeam/greptimedb/pull/4530/commits/73ac3b14788ee06b87c84bb5800e902687120a30)